cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/model/IncentivePublicParameters.java

```diff
@@ -61,7 +61,7 @@
     SetMembershipPublicParameters eskBaseSetMembershipPublicParameters;
 
     @NonFinal
-    int numEskDigits; // This is computed in the init method since it contains redundant data
+    int numEskDigits; // rho from the 2020 inc sys paper, this is computed in the init method since it contains redundant data
```

Comment:
inc sys paper could be denoted as [incsys20] and then mention is at the class level comment.

Reply:
also you could write what rho should do here

Reply:
I assumed that someone reading our code knows about the mentioned paper and thus gets the reference, we could maybe link it in the projects main README.
Added some explanation about rho.